extensions/sql-migration/src/dialog/configureIR/configureIRDialog.ts

```diff
@@ -188,7 +188,7 @@ export class ConfigureIRDialog {
 		}).component();
 
 		// get the SHIR script
-		const scriptPath = path.join(__dirname, '../../../scripts/SHIR-auto-configuration.ps1');
+		const scriptPath = path.join(__dirname, '../../scripts/SHIR-auto-configuration.ps1');
```

Comment:
What is root directory from where we are trying this relative path? is it "\.azuredatastudio-rc1\extensions\microsoft.sql-migration-1.5.4\migrationService\Windows\4.12.0.4"?